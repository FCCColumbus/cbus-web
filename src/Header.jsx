
import links from './links'
// TODO: add permanant?? discord link

function Header() {

    return (
        <header>
            <nav>
                <ul>
                    {links.map(link => <li key={link.text}><a href={link.href}>{link.text === 'Home' ? <img src='/fcc-logo.svg' alt='fcc logo' className='header-logo' /> : link.text}</a></li>)}
                </ul>
            </nav>
            <h1>free code camp
                <img src='free-code-camp-logo.svg' alt='fcc logo' className='big-logo' /> columbus</h1>
        </header>
    )
}

export default Header