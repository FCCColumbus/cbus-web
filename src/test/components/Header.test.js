import renderer from "react-test-renderer";

import Header from "../../components/Header";

describe("Header", () => {
  it("renders", () => {
    const view = renderer.create(<Header />);
    expect(view).toMatchSnapshot();
  });
});