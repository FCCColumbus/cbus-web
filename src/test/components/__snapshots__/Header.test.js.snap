// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<header>
  <nav>
    <ul>
      <li>
        <a
          href="/"
        >
          <img
            alt="fcc logo"
            className="header-logo"
            src="/fcc-logo.svg"
          />
        </a>
      </li>
      <li>
        <a
          href="#about"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="https://www.freecodecamp.org/"
        >
          FreeCodeCamp
        </a>
      </li>
      <li>
        <a
          href="https://discord.gg"
        >
          Discord
        </a>
      </li>
      <li>
        <a
          href="https://github.com/FCCColumbus"
        >
          GitHub
        </a>
      </li>
      <li>
        <a
          href="#events"
        >
          Events
        </a>
      </li>
      <li>
        <a
          href="https://github.com/FCCColumbus/cbus-web/graphs/contributors"
        >
          Contributors
        </a>
      </li>
    </ul>
  </nav>
  <h1>
    free code camp
    <img
      alt="fcc logo"
      className="big-logo"
      src="free-code-camp-logo.svg"
    />
     columbus
  </h1>
</header>
`;
